### PR TITLE
Downgrade the bootstrap version

### PR DESCRIPTION
**Pull request by ahmed-tranzito**
Downgrade the bootstrap version from `5.2.2` to `4.5.2`

### CODE REVIEW

File: app/views/tranzito_utils/_flash_messages.html.haml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     - flash.each do |type, message|
       - next unless message.present?
       .alert.alert-dismissible.fade.show{class: "alert-#{type}"}
-        %button.btn-close{"data-bs-dismiss" => "alert", "aria-label" => "Close"}
+        %button.btn-close{"data-dismiss" => "alert", "aria-label" => "Close"}
```
Comment (sethherr) [inline app/views/tranzito_utils/_flash_messages.html.haml]: This needs to change to `.close` and needs to have an `x` content

Check the partial in bikehub_webapp

File: spec/dummy/app/views/home/index.html.haml [hunk 2]
```diff
@@ -6,8 +6,6 @@
     %title= page_title # page_title is defined in Helpers
     = csrf_meta_tags
     = csp_meta_tag
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
```
Comment (sethherr) [inline spec/dummy/app/views/home/index.html.haml:9]: After this, the dummy app doesn't include any bootstrap styles. It probably should

**Review by sethherr — CHANGES_REQUESTED** [review]
2 things

File: app/views/tranzito_utils/_flash_messages.html.haml (hunk 1 above)
Comment (ahmed-tranzito) [reply app/views/tranzito_utils/_flash_messages.html.haml]: Yeah I noticed too and I added it, let me push it.

**Review by sethherr — COMMENTED** [review]
👍

**Review by sethherr — APPROVED** [review]

**Pull request merged**